www/js/overlay.js

```diff
@@ -314,7 +326,16 @@ textControls.append(separatorY());
 textControls.append(controlsFactory(changeBarSizeButton, 'Size'));
 textControls.append(controlsFactory(changeOpacityButton, 'Opacity'));
 textControls.append(separatorY());
-textControls.append(controlsFactory([topLayoutBtn, bottomLayoutBtn, splitLayoutBtn], 'Layout'));
+textControls.append(
+  controlsFactory(
+    [topLayoutBtn, bottomLayoutBtn, splitLayoutBtn, verticalLayoutBtn, classicLayoutBtn],
+    'Layout',
+  ),
+);
+
+document
+  .querySelector('.layout-btn.vertical')
+  .classList.toggle('vertical-left', overlayVars.layout === 'vertical-left');
```

Comment:
think this needs to be flipped. The icon should be for the opposite of what is being shown so it indicates there another option.